client/jetpack-onboarding/style.scss

```diff
@@ -13,11 +13,63 @@
 		}
 	}
 
+	.steps__summary-columns {
```

Comment:
I'm thinking we can keep the CSS here more minimal and rely less on static values. Considering the recommendations above, I think the following CSS should be sufficient to cover all cases:

```
.steps__summary-columns {
	display: flex;
	justify-content: center;
	font-size: 14px;

	@include breakpoint( '<480px' ) {
		display: block;
		text-align: center;
	}
}

.steps__summary-column {
	margin: 0 30px 20px;

	@include breakpoint( '<480px' ) {
		display: inline-block;
		text-align: left;
	}
}

.steps__summary-heading {
	font-weight: bold;
	margin-bottom: 20px;
}

.steps__summary-entry {
	margin-bottom: 20px;

	.gridicon {
		position: relative;
		top: 3px;
		margin-right: 12px;
	}

	&.completed {
		color: $gray;
	}

	&.todo {
		color: $blue-medium;
	}
}
```

This would look like this:

Desktop:
![](https://cldup.com/chktGXuI3K.png)

Mobile:
![](https://cldup.com/3eZcBHD-jT.png)

Note: we're departing from the original mockups a little here, alignment-wise (centering stuff vs keeping the original design positioning), but personally I think it's quote beneficial, because we're staying as consistent as possible with the rest of Calypso styles, while we're still flexible with the content that's inside. I think we could also use some feedback from @MichaelArestad on this one.

Reply:
IMHO the horizontal layout make things much more readable for mobile 👍